daemons/controld/controld_te_callbacks.c

```diff
@@ -515,6 +515,7 @@ te_update_diff_v2(xmlNode *diff)
                      op, xpath, name);
         }
     }
+    controld_clear_outside_event_table();
```

Comment:
We might as well do this in te_update_diff() so it happens regardless of diff format, mainly for future-proofing against a theoretical v3 since v1 should only be possible in a rolling upgrade from 1.1.11.